sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -147,6 +150,11 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
     this._context = context;
     this.entityPath = this._context.entityPath;
     this._sessionReceiverOptions = _sessionOptions;
+    if (this._sessionReceiverOptions.retryOptions) {
+      this._sessionReceiverOptions.retryOptions.timeoutInMs = getRetryAttemptTimeoutInMs(
```

Comment:
Is this a good function to call? Looking at it it looks like if you specify anything < 60 seconds it just automatically bumps it up to 60 seconds.

https://github.com/ramya-rao-a/azure-sdk-for-js/blob/c1a0e44e43bdb7115820c7fafca30556183ba4f1/sdk/servicebus/service-bus/src/util/utils.ts#L41

Is this intentional?

Reply:
Logged #8193 for further investigations, but using it right now to mirror what we do in Event Hubs